dart-impl/tasking/src/internal/dart_tasking_phase.c

```diff
@@ -206,6 +205,7 @@ dart__tasking__phase_add_task()
     int32_t val = DART_FETCH_AND_INC32(
                     &phase_task_counts[creation_phase%max_active_phases_mod]);
     if (val == 0) {
+      DART_FETCH_AND_INC32(&num_active_phases);
```

Comment:
That seems to be a conflict (the same change exists in the base branch). Can you please rebase?